src/store/normalize-data.js

```diff
@@ -79,6 +79,11 @@ export const createInitialPipelineState = () => ({
     active: {},
     enabled: {},
   },
+  slice: {
+    from: null,
+    to: null,
+    apply: false,
```

Comment:
Does apply indicate whether from and to are not null or is there any other reason for storing this key ? Are we using apply for other reasons in the store ?

Reply:
`apply` will help the flowchart know when to actually render just the slicedPipeline or not, it does not affect whether from and to are null or valid